app/bundles/CoreBundle/Doctrine/Type/ArrayType.php

```diff
@@ -39,6 +39,10 @@ function (&$entry) {
 
         $serialized = serialize($value);
 
+        if(strpos($serialized, chr(0)) !== false) {
```

Comment:
Btw this is bug of Doctrine ArrayType (not Mautic). I'll open PR also there. 

-----

Example when replacing null with nothing is wrong.

```php
class A {private $b;}

$s = str_replace("\0", "", serialize(new A));
$o = deserialize($s); // error

// serialized object has diferrent structure -> cannot deserialize
class A {public $Ab;} 
```